SCMS.Services.Api/Migrations/20220222181518_InitializeModels.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace SCMS.Services.Api.Migrations
 {
-    public partial class ReInitializeModels : Migration
+    public partial class InitializeModels : Migration
```

Comment:
You need to delete the Migrations folder every time and re-create.

Reply:
done